nncf/common/graph/graph.py

```diff
@@ -32,6 +31,13 @@
 LayerName = str
 
 
+def get_edge_boundaries(match: List[str], graph: nx.DiGraph):
```

Comment:
Docstring, return type hint

Reply:
Done

Reply:
Making this a nested function reduces readability. Move this to a private method of NNCFGraph instead.

Reply:
should I make it static in that case?

Reply:
Done